src/components/app/types.ts

```diff
@@ -1,186 +1,200 @@
-import { RouteComponentProps } from 'react-router'
-import { Teams } from '../../services/service.types'
-import { DeploymentAppType } from '../v2/appDetails/appDetails.type'
+import { RouteComponentProps } from 'react-router';
```

Comment:
Seems like some issue with formatter

Reply:
It added semi-colons but we don't allow this in auto format rules. Please check.

Reply:
Done